src/elf.rs

```diff
@@ -525,8 +490,6 @@ impl<C: ContextObject> Executable<C> {
                 Section::Owned(_, data) => data.capacity(),
                 Section::Borrowed(_, _) => 0,
             })
```

Comment:
Should we include an arm like this to account for a possible owned data?

Reply:
The text section is not supposed to ever be owned / copied. Hence why all `match` expressions against it have an `unreachable!` arm.